Upstream-Status: Submitted [https://github.com/pytest-dev/pytest-html/pull/516]
Signed-off-by: Ross Burton <ross.burton@arm.com>

From d7fbc79d428323c838ce68a51e1e1806c2096a76 Mon Sep 17 00:00:00 2001
From: Ross Burton <ross.burton@arm.com>
Date: Mon, 23 May 2022 13:46:50 +0100
Subject: [PATCH] pip isn't a build dependency

It's possible, and even desirable, to build modern Python code without pip, for example by using the `build` and `installer` packages.
---
 pyproject.toml | 1 -
 1 file changed, 1 deletion(-)

diff --git a/pyproject.toml b/pyproject.toml
index 2683f072..e3f25bf7 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -1,6 +1,5 @@
 [build-system]
 requires = [
-  "pip >= 19.3.1",
   "setuptools >= 42",
   "setuptools_scm[toml] >= 3.5.0",
   "setuptools_scm_git_archive >= 1.1",
